src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java

```diff
@@ -993,351 +993,6 @@ public void testCreateGeneratorInBusBreaker() throws Exception {
-                .voltageLevelId1("v1")
-                .busOrBusbarSectionId1("1.1")
-                .voltageLevelId2("v2")
-                .busOrBusbarSectionId2("1A")
-                .magnetizingConductance(100.0)
-                .magnetizingSusceptance(200.0)
-                .ratedVoltage1(1000)
-                .ratedVoltage2(1010)
-                .seriesReactance(300)
-                .seriesResistance(400)
-                .connectionName1("cnid2wt1")
-                .connectionDirection1(ConnectablePosition.Direction.TOP)
-                .connectionName2("cnid2wt2")
-                .connectionDirection2(ConnectablePosition.Direction.TOP)
-                .ratioTapChanger(ratioTapChangerCreationInfos)
-                .build();
-        testCreateTwoWindingsTransformerInNodeBreaker(twoWindingsTransformerCreationInfos);
-    }
-
-    @Test
-    public void testCreateTwoWindingsTransformerWithPhaseTapChangerInNodeBreaker() throws Exception {
-        final String transformerId = "id2wt1WithPhaseTapChanger";
-        // create new 2wt in voltage level with Node/breaker topology, having a PhaseTapChanger
-        PhaseTapChangerCreationInfos phaseTapChangerCreationInfos = PhaseTapChangerCreationInfos.builder()
-                .lowTapPosition(0)
-                .tapPosition(1)
-                .regulating(true)
-                .targetDeadband(null)
-                .regulationMode(PhaseTapChanger.RegulationMode.ACTIVE_POWER_CONTROL)
-                .regulationValue(10.0)
-                .regulatingTerminalVlId("v1")
-                .regulatingTerminalId("v1load")
-                .regulatingTerminalType("LOAD")
-                .steps(getTapChangerSteps())
-                .build();
-        TwoWindingsTransformerCreationInfos twoWindingsTransformerCreationInfos = TwoWindingsTransformerCreationInfos.builder()
-                .type(ModificationType.TWO_WINDINGS_TRANSFORMER_CREATION)
-                .equipmentId(transformerId)
-                .equipmentName("2wtName")
-                .voltageLevelId1("v1")
-                .busOrBusbarSectionId1("1.1")
-                .voltageLevelId2("v2")
-                .busOrBusbarSectionId2("1A")
-                .magnetizingConductance(100.0)
-                .magnetizingSusceptance(200.0)
-                .ratedVoltage1(1000)
-                .ratedVoltage2(1010)
-                .seriesReactance(300)
-                .seriesResistance(400)
-                .connectionName1("cnid2wt1")
-                .connectionDirection1(ConnectablePosition.Direction.TOP)
-                .connectionName2("cnid2wt2")
-                .connectionDirection2(ConnectablePosition.Direction.TOP)
-                .phaseTapChanger(phaseTapChangerCreationInfos)
-                .build();
-        testCreateTwoWindingsTransformerInNodeBreaker(twoWindingsTransformerCreationInfos);
-    }
-
-    @Test
-    public void testCreateTwoWindingsTransformerInMixedTopology() throws Exception {
```

Comment:
I cant find this test anymore.
the question is: where do we have to put it ?
I'm afraid it is using a third network ...